Product/Product.css

```diff
@@ -62,7 +62,23 @@
     margin-left:0px;
     font-size: 14px;
 }
-
+.price-input{
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
```

Comment:
Someone in my PR review suggested me that if you have measurements less than 1 rem then use them but I have not looked into it yet so just passing down the suggestion to you.

Reply:
Thanks